libs/hmpb/render-backend/src/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from './all_bubble_ballot_fixtures';
 export * from './ballot_fixtures';
-export * from './render_ballot';
+export * from './next';
```

Comment:
No, I'm planning to combine `hmpb/layout` and `hmpb/render-backend` into one library, since there's no need to run the layout code in the browser. Will move `next` to the top level at that point. Tracking follow ups here: https://github.com/votingworks/vxsuite/issues/4559